### PR TITLE
introduces melos as workspace management and clean up of the templates

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
As a fan of cargo workspace in rust, I would like to have in the env the Melos that takes care of all the packages and make life easier

### CODE REVIEW

File: examples/starwars/pubspec.yaml [hunk 1]
```diff
@@ -9,17 +9,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  graphql_flutter:
-  graphql:
+  graphql_flutter: 5.0.1
+  graphql: 5.0.1
```
Comment (budde377) [inline examples/starwars/pubspec.yaml:13]: Should this be paths? 🤔

**Comment by budde377** [comment]
Looks great! Do you want to setup the actions to use melos in this PR?

**Comment by vincenzopalazzo** [comment]
>Do you want to setup the actions to use melos in this PR?

Mh, in this case, melos can fix the linking between packages in the Ci right? maybe I need to develop some custom scrips to divided the test in graphql_flutter and graphql and keep the actual configuration.

Maybe this can be done in another PR?

What do you think?

**Comment by budde377** [comment]
It's up to you! I think it would be fairly easy but if you prefer to split it, that's fine aswell.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ labels: 'enhancement'
 assignees: ''
 
 ---
-
+<!--
```
Comment (budde377) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]: What are your thoughts on removing these?

File: .github/workflows/graphql_build.yml [hunk 3]
```diff
@@ -21,5 +21,5 @@ jobs:
       - name: Code analyze check
         run: |
           cd packages/graphql
-          dart analyze lib
-          dart analyze test
\ No newline at end of file
+          dart analyze lib --fatal-infos
+          dart analyze test --fatal-infos
```
Comment (budde377) [inline .github/workflows/graphql_build.yml:25]: Can you add some context on why we're doing this?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (vincenzopalazzo) [reply .github/ISSUE_TEMPLATE/feature_request.md:9]: I would like to have the content as a comment and not as a template, so the user could uncomment only the part that they want. I'm missing some things?

File: .github/workflows/graphql_build.yml (hunk 3 above)
Comment (vincenzopalazzo) [reply .github/workflows/graphql_build.yml:25]: for some reason, we are missing dart code error and the test doesn't fail, this increases the level of failure, do you think that it is useful?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (budde377) [reply .github/ISSUE_TEMPLATE/feature_request.md:9]: Thanks, makes sense! I thought it would just remove the message m, but a comment sounds good

**Review by budde377 — APPROVED** [review]

**Pull request merged**